doc/playbook.rst

```diff
@@ -248,8 +248,62 @@ A few useful commands for managing the service:
 * ``kubectl logs <pod>`` outputs the entire log associated with a particular pod.
   This can be a long file, so consider piping to ``less`` or ``grep``.
   ``kubectl logs`` also offers the ``-f`` flag for streaming output.
-* ``kubectl describe <pod>`` lists the entire configuration of a particular pod.
-  One useful entry is ``Containers:user-container:Image``, which gives `the hash of the service container <https://github.com/lsst-dm/prompt_prototype/pkgs/container/prompt-proto-service>`_ running on that pod and can be used to infer which version of the code is being tested.
+
+Troubleshooting
+^^^^^^^^^^^^^^^
+
+Deleting Old Services
+"""""""""""""""""""""
+
+Normally, old revisions of a service are automatically removed when a new revision is deployed.
+However, sometimes an old revision will stick around; this seems to be related to Python errors from bad code.
+Such revisions usually manifest as a "CrashLoopBackOff" pod in ``kubectl get pods``.
+
+To delete such services manually:
+
+.. code-block:: sh
+
+   kubectl get revision  # Find the name of the broken revision
+   kubectl delete revision <revision name>
+
+.. note::
+
+   There's no point to deleting the pod itself, because the service will just recreate it.
+
+Identifying a Pod's Codebase
+""""""""""""""""""""""""""""
+
+To identify which version of ``prompt-prototype`` a pod is running, run
+
+.. code-block:: sh
+
+   kubectl describe pod <pod name> | grep "prompt-proto-service@"
+
+This gives the hash of the service container running on that pod.
+Actually mapping the hash to a branch version may require a bit of detective work; `the GitHub container registry <https://github.com/lsst-dm/prompt_prototype/pkgs/container/prompt-proto-service>`_ (which calls hashes "Digests") is a good starting point.
+
+The version of Science Pipelines used can be hard to identify by the above method, because it depends on the exact build order of the two containers.
```

Comment:
Here is another way to identify the stack version. 

Going to the page of a specific container (for example  [this old one](https://github.com/lsst-dm/prompt_prototype/pkgs/container/prompt-proto-service/63253721?tag=DM-36586) ), in its manifest, the `labels`  / `EUPS_TAG` field in the bottom shows the stack version from the base container.

Reply:
Thanks, that's a much better way!